Elegance Station

IAmNotGray opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Elegance, or "Lady Elegance" in full, is a service-oriented station somewhere in between the size of Submarine and Hammurabi with a minimum player count of 40.
In addition to this, Elegance features its own unique evac shuttle, dubbed the "NTES Transport."
(Original got closed because of branch stuff.)
## Why / Balance
I like the idea of a service-oriented station, and I haven't seen anyone else attempt to do anything similar to this.
_I also couldn't help myself as a service main..._
## Media
![Elegance-0](https://github.com/user-attachments/assets/80982521-9eb6-4a97-b31b-278cd992e4e0)
Elegance (Outdated and too large to update the image)
![image](https://github.com/user-attachments/assets/14b24a78-ed20-43cf-989c-8e62833686f8)
NTES Transport

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: Elegance, a service station, is now recieving crew members!
- add: The new NTES Transport is here to take you to Central Command. _In style!_

IAmNotGray committed:
Elegance Time..

IAmNotGray committed:
Elegance Evac & Job List Update

It's so real.

IAmNotGray committed:
Actual Elegance Job Slot Changes

IAmNotGray committed:
Potential Invalids Fix???-

IAmNotGray committed:
Elegance Last(?) Changes

Second to last, if not the last changes to Elegance for a good while.
We're so back!

IAmNotGray committed:
Oops :3

IAmNotGray committed:
I'm Trying

Fixed some wiring and lighting issues at the bridge.

IAmNotGray committed:
Merge branch 'master' of https://github.com/IAmNotGray/Delta-v

IAmNotGray committed:
More changes

Ditto

IAmNotGray committed:
Invalids 2: Electric Boogalo

Hopefully this actually works. If not, I'm going to.. do something, for sure-

HUGE thanks to Velcro for helping me with this one!!

IAmNotGray committed:
Removed the paper slip from Transport.

It was causing invalid errors...

deltanedas commented:
some things i noticed from looking at the render

engi
- teg has a bunch of manual valves you would only use to sabotage it, particularly the cold loop
- burn chamber doors dont use logic gates, they will get stuck open at some point by a clueless atmos or ce guaranteed, make them either single doors or use logic gates to have them bolt when either door is open (just linking the doors directly will break)

botany:
- no botanist lockers
- no biogenerator
- no chemmaster

freezer:
- map render may be trolling but very suspicious that theres just an air vent with no passive vent for the freezer

epi:
- there is next to no gas control for the artifact chambers, just space blast doors and thats it

Avalon-Proto commented:
Personally my biggest gripe is the fact the sec hardsuits are in the part of the armory that's most likely to get spaced. So the sec armored hardsuits are likely to be lost the moment nukies hit

FieldCommand commented:
Its an interesting station, but my gripe is the maintenance on this station. They are huge, mostly double wide corridors or sometimes triple or more. And they are very uniform/monotone, with hardly any clutter in them, few to no lockers/wall lockers, or interesting sightings and ester eggs.

IAmNotGray commented:
> some things i noticed from looking at the render
> 
> engi
> 
> * teg has a bunch of manual valves you would only use to sabotage it, particularly the cold loop
> * burn chamber doors dont use logic gates, they will get stuck open at some point by a clueless atmos or ce guaranteed, make them either single doors or use logic gates to have them bolt when either door is open (just linking the doors directly will break)
> 
> botany:
> 
> * no botanist lockers
> * no biogenerator
> * no chemmaster
> 
> freezer:
> 
> * map render may be trolling but very suspicious that theres just an air vent with no passive vent for the freezer
> 
> epi:
> 
> * there is next to no gas control for the artifact chambers, just space blast doors and thats it

On the burn chamber doors, I can go ahead and fix that very easily.
And about the freezer, I didn't think it'd matter if there was a passive vent or a normal air vent. I've gone ahead and changed that.
As for Epistemics, I've tried to fit in a way to add gas to the chambers, there's just not enough. If you saw the steel, though, that's for ol' John Research Assistant to set it up!

Gone ahead and touched up everything else you've adressed as well.

IAmNotGray commented:
> Personally my biggest gripe is the fact the sec hardsuits are in the part of the armory that's most likely to get spaced. So the sec armored hardsuits are likely to be lost the moment nukies hit

I've moved some things around.

IAmNotGray commented:
> Its an interesting station, but my gripe is the maintenance on this station. They are huge, mostly double wide corridors or sometimes triple or more. And they are very uniform/monotone, with hardly any clutter in them, few to no lockers/wall lockers, or interesting sightings and ester eggs.

I made the decision to make the maintenance tunnels this way pretty early on, so I don't remember WHY I did it exactly, but I'm pretty sure I've been told by several people that big maints are pretty neat?-

FieldCommand commented:
> I made the decision to make the maintenance tunnels this way pretty early on, so I don't remember WHY I did it exactly, but I'm pretty sure I've been told by several people that big maints are pretty neat. -

I don't mind big maintenance, but this is a bit excessive. All this space, and there is barely anything going on here. You have a disposal room, you should move the recycler out of the corner. and you have a secret room, the syndicate baner can be seen from outside and should be replaced with a free-standing one. And for how big this is, its very monotonous. There are some lockers and some canisters, but not all of them are at the access points. When an emergency arises, I would have to walk through all this to find an N2 or O2 locker. and I don't see any wall lockers at all. And this should also be partitioned into sections with firelocks, it has a lot of volume, and spacing would be a disaster here and take for ever to fill again.

![Capture10](https://github.com/user-attachments/assets/59e75a6a-cbee-4852-85cf-10c73176a243)

IAmNotGray commented:
> > I made the decision to make the maintenance tunnels this way pretty early on, so I don't remember WHY I did it exactly, but I'm pretty sure I've been told by several people that big maints are pretty neat. -
> 
> I don't mind big maintenance, but this is a bit excessive. All this space, and there is barely anything going on here. You have a disposal room, you should move the recycler out of the corner. and you have a secret room, the syndicate baner can be seen from outside and should be replaced with a free-standing one. And for how big this is, its very monotonous. There are some lockers and some canisters, but not all of them are at the access points. When an emergency arises, I would have to walk through all this to find an N2 or O2 locker. and I don't see any wall lockers at all. And this should also be partitioned into sections with firelocks, it has a lot of volume, and spacing would be a disaster here and take for ever to fill again.
> 
> ![Capture10](https://private-user-images.githubusercontent.com/159087063/377587915-59e75a6a-cbee-4852-85cf-10c73176a243.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkxOTA0NDcsIm5iZiI6MTcyOTE5MDE0NywicGF0aCI6Ii8xNTkwODcwNjMvMzc3NTg3OTE1LTU5ZTc1YTZhLWNiZWUtNDg1Mi04NWNmLTEwYzczMTc2YTI0My5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxN1QxODM1NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMjMzNTZiYzAyNGVlY2UxYjAzN2M4ZTQwYTQ5MDU0OTgxYWM0ZTRkMzdjOTNiYjJlNWQ2NWFhZWE1MzJhMzM5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-2o1O3Scnue08K0Meebbc1X0fCSDN63aqPbxTX3OSgA)

Oh, yeah, I see exactly what you mean here. Yikes, I might have forgotten about that area. I'll be spending a lot of time touching up maints...
I was honestly scared to clutter things up too much and make maints hard to navigate.

IAmNotGray committed:
Elegance Maints Update

Greatly improves the maintenance tunnels on the station, as wells as fixes some minor issues.

IAmNotGray committed:
Merge branch 'master' into Elegance

IAmNotGray commented:
I've ended up addressing the issues brought up with the station with the update here. Mostly the maintenance tunnels, but I've also ended up moving around the security suits in the armory, among other small things.

IAmNotGray committed:
Another Small Update

Fixed some complaints with Chemistry, the kitchen, and added some goodies to the outside of the station.

And trash. A lot of trash.

IAmNotGray committed:
Merge branch 'Elegance' of https://github.com/IAmNotGray/Delta-v into Elegance

IAmNotGray committed:
Minor Changes

The Space View in med is so real.
Also forgot to color some pipes...

IAmNotGray committed:
More Tweaks

Added a toilet to Perma, among other minor changes, and some prototype file changes.

IAmNotGray committed:
Even More Tweaks

There were a few wiring issues, so I fixed those. Also some very minor maints updates, along side some decoration.

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Armory Changes

The entire armory got an overhaul, to sum this commit up. Not much changed other than this besides the salvage magnet's orientation.

IAmNotGray committed:
Changes... Again.

Fixed some minor oopsies and oversights.

I'm also calling this one the fill-in-ing. Cluttered various rooms because they were way too empty for their own good.

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Actually made the updates this time

I'm stupid, to sum this commit up...

IAmNotGray committed:
Epistemics Rework

COMPLETELY overhauled this fucking department. It was terrible and empty before and I hated it.

IAmNotGray committed:
Stupid Fucking Grille

IAmNotGray commented:
I completely overhauled Epistemics. A big issue with this station was the amount of unused space and hopefully I fixed that for the most part.

IAmNotGray committed:
Botany Overhaul

Botany has received a major overhaul. I've also added late join spawn points and cryopods because I forgot to do this. Rejoice.

IAmNotGray committed:
Final Final Touch-ups

Real this time, I swear.

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
I lied.

Who's surprised?

IAmNotGray committed:
Curse of EVERYTHING

God this was fucking abysmal but I DID IT

IAmNotGray commented:
Okay. That's hopefully the last update I'm doing until Elegance gets reviewed. Thanks to everyone for all the feedback!

Kr8art commented:
Love this! The shuttle looks great too!

IAmNotGray committed:
That Fucking Thing That I Hate

IAmNotGray committed:
The Greening of '24

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
The Christmas Update

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Update elegance.yml

IAmNotGray committed:
Elegance Assistants

IAmNotGray committed:
Head Bedroom Updates

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Elegance Hotfixes

IAmNotGray committed:
More hotfixes

IamVelcroboy commented on Content.IntegrationTests/Tests/PostMapInitTest.cs:
```diff
@@ -62,6 +62,7 @@ public sealed class PostMapInitTest
             "Lighthouse", // DeltaV
             "Micro", // DeltaV
             "Chibi" // DeltaV
```
need a comma

IAmNotGray committed:
Elegance Review Fixed 2/?

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
I'm going fucking nuts

IAmNotGray committed:
Merge branch 'Elegance' of https://github.com/IAmNotGray/Delta-v into Elegance

IAmNotGray committed:
Update elegance.yml

IAmNotGray committed:
Merge branch 'DeltaV-Station:master' into Elegance

IAmNotGray committed:
Moar...

IAmNotGray committed:
Update elegance.yml

IAmNotGray committed:
Update elegance.yml

IamVelcroboy commented:
![image](https://github.com/user-attachments/assets/c3541d06-6d9f-40b7-92ad-9ef901fc59d7)

IAmNotGray commented:
> ![image](https://private-user-images.githubusercontent.com/107660393/408767249-c3541d06-6d9f-40b7-92ad-9ef901fc59d7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzgzODY0NDgsIm5iZiI6MTczODM4NjE0OCwicGF0aCI6Ii8xMDc2NjAzOTMvNDA4NzY3MjQ5LWMzNTQxZDA2LTZkOWYtNDBiNy05MmFkLTllZjkwMWZjNTlkNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIwMVQwNTAyMjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYjIwODYwMDk1ZTAzNDQyOTcyNTkyZTY0MjBiZDNhYzIyNjQzYzE1NjZlMjNlMjk0OWU2YTcxZjk1MTQwZmU5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.yMgasV0FBikBOwz6qDjc_Nn0XIwL5b6g7R0NipPChEQ)

Oh dear god.

IAmNotGray committed:
Added Salvage Spawns

IAmNotGray committed:
Review Fixes (1/3)

IAmNotGray commented:
Oh, I blundered the commit name. (3/3)*
:blunt:

IAmNotGray committed:
Merge branch 'master' into Elegance

IAmNotGray committed:
Velcro's lighting fix!!

It's here!~

IAmNotGray commented:
Gamers, it's ready.

IAmNotGray committed:
Update NTES_Transport.yml

IamVelcroboy commented on Resources/Maps/elegance.yml:
```diff
@@ -85768,6 +85871,22 @@ entities:
     - type: Transform
       pos: -28.477945,33.979656
       parent: 2
+    - type: GroupExamine
```
Dang, I did a thing. These lines need to be removed

IamVelcroboy replied:
lines 85874-85889

IamVelcroboy commented on Resources/Maps/elegance.yml:
```diff
@@ -27315,35 +27327,13 @@ entities:
-      rot: -1.5707963267948966 rad
-      pos: 24.5,49.5
-      parent: 2
 - proto: AlwaysPoweredLightColoredFrostyBlue
```
Missed this always powered

IamVelcroboy commented on Resources/Maps/elegance.yml:
```diff
@@ -27315,35 +27327,13 @@ entities:
-      rot: 1.5707963267948966 rad
-      pos: 76.5,23.5
-      parent: 2
 - proto: AlwaysPoweredSmallLightMaintenance
```
and this one

IAmNotGray committed:
Update elegance.yml
